handbook/product/product_management/direction_template.md

```diff
@@ -0,0 +1,75 @@
+# <Product> direction
```

Comment:
```suggestion
# Vision, Strategy, and Roadmap Template

> ℹ️  The following is a template for use on the pages of specific product & engineering teams, used to record and share the vision, strategy, and roadmap.<br>Copy to your team's page and complete accordingly.

<hr>

# <Product> direction
```

Reply:
Think it's useful to make it super clear that this is a template - it should be obvious, but just in case 😁

Reply:
Was outdated by one of muy commits, but incorporated it and pushing now.